pkg/ansible/controller/reconcile.go

```diff
@@ -183,7 +182,8 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 			}
 		}
 		if event.Event == eventapi.EventRunnerOnFailed && !event.IgnoreError() {
-			failureMessages = append(failureMessages, event.GetFailedPlaybookMessage())
+			eventErr := errors.New("event runner on failed")
+			return reconcile.Result{}, eventErr
```

Comment:
I think we still want the markDone function to be called in this case (if status management is enabled), that's what puts the failure message in the status reflects that a run did not succeed. We should do this even if we requeue after

Reply:
https://github.com/operator-framework/operator-sdk/pull/2375/files#diff-9649888b3df1961dad649345545de4c0R304

Reply:
Thank you for your review @fabianvf. 
Done. Wdyt now?